__tests__/pages/resource/id/evaluate.test.tsx

```diff
@@ -1,29 +1,129 @@
-import { render, screen, act } from "@testing-library/react";
+import { render, screen, act, fireEvent, within } from "@testing-library/react";
 import "@testing-library/jest-dom";
-import { createMockRouter } from "../../../helpers/testHelpers";
+import {
+  mantineRecoilWrap,
+  createMockRouter,
+  getMockFetchImplementation,
+  getMockFetchImplementationError,
+} from "../../../helpers/testHelpers";
 import { RouterContext } from "next/dist/shared/lib/router-context";
 import EvaluateMeasurePage from "../../../../pages/[resourceType]/[id]/evaluate";
+import { DateTime } from "luxon";
+
+const MEASURE_BODY_WITH_DATES = {
+  resourceType: "Measure",
+  id: "measure-EXM104-8.2.000",
+  meta: {
+    lastUpdated: "2022-07-21T18:12:25.008Z",
+  },
+  effectivePeriod: {
+    start: "2019-01-01",
+    end: "2019-12-31",
+  },
+};
+
+const MEASURE_BODY_NO_DATES = {
+  resourceType: "Measure",
+  id: "measure-EXM104-8.4.000",
+  meta: {
+    lastUpdated: "2022-07-21T18:12:25.008Z",
+  },
+  effectivePeriod: {},
+};
+
```

Comment:
mayhaps we should add one last test case for `effectivePeriod` missing entirely since 'tis 0..1 in the FHIR spec? Assertions should be the same as the one with a blank effectivePeriod, but worth testing anyway